packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -58,7 +58,7 @@
             <details-and-edit
               :is-folder-share="highlightedFile.isFolder"
               :link="link"
-              :modifiable="canEdit"
+              :modifiable="canEdit && !isLinkIndirect(link)"
```

Comment:
Wouldn't `:modifiable="canEdit && !link.indirect` already do the same without the additional method? 🤔

Reply:
True, let me open a quick PR bumping deps and simplifying this